packages/provider/src/language-model/v2/index.ts

```diff
@@ -10,7 +10,6 @@ export * from './language-model-v2-logprobs';
 export * from './language-model-v2-prompt';
 export * from './language-model-v2-provider-defined-tool';
 export * from './language-model-v2-provider-metadata';
-export * from './language-model-v2-provider-options';
```

Comment:
maybe i should add an alias we can later remove?

Reply:
no, lets keep different spec (embedding, languagemodel, etc) fully separate for now, also across versions. it's only types so no runtime overhead, and it lets us change one spec without impacting other specs